.eslintrc.json

```
@@ -1,68 +1,56 @@
 {
-    "env": {
-        "browser": true,
-        "es2022": true,
-        "node": true,
-        "jest": true
-    },
-    "extends": [
-        "plugin:jsdoc/recommended",
-        "eslint:recommended",
-        "plugin:@typescript-eslint/eslint-recommended",
-        "plugin:@typescript-eslint/recommended",
-        "next/core-web-vitals",
-        "next"
+  "env": {
+    "browser": true,
+    "es2022": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": [
+    // "plugin:jsdoc/recommended",
```

Disabling these rules for now since they seem to be flooding the ESLint results and hiding the errors.

---

```suggestion
    // Disabled until we actually resolve most of the warnings it points out,
    // to avoid warning blindness:
    // "plugin:jsdoc/recommended",
```

Also: I've started to also dislike that JSON doesn't support comments.